src/Mod/TechDraw/Gui/TaskBalloon.ui

```diff
@@ -358,6 +358,19 @@
      </item>
     </layout>
    </item>
+   <item>
+    <spacer name="verticalSpacer">
+     <property name="orientation">
+      <enum>Qt::Vertical</enum>
+     </property>
+     <property name="sizeHint" stdset="0">
+      <size>
+       <width>20</width>
+       <height>40</height>
```

Comment:
The height changes are done by QtDesigner.  The layout of the useful widgets doesn't change as the spacer pushes the other widgets up.    The spacer prevents the ugly situation in the first screenshot if a second task dialog is opened and then closed.  No functional impact.  

![withoutSpacer](https://user-images.githubusercontent.com/3587161/172388267-2b38c589-6589-4e09-8ecf-1e6a54f40d1f.png)
![withSpacer](https://user-images.githubusercontent.com/3587161/172388272-52ff6626-cc08-43e9-8cd4-8ace828c0ee8.png)

Reply:
> The height changes are done by QtDesigner. The layout of the useful widgets doesn't change as the spacer pushes the other widgets up. The spacer prevents the ugly situation in the first screenshot if a second task dialog is opened and then closed. No functional impact.

- your screenshots shows that there is always too much vertical space, with the spacer you only shift it
- I see that the height is fixed, which should not be the case. I fixed this now: https://github.com/FreeCAD/FreeCAD/commit/18eb44d120c9e
Does this fix the issue for you? 
- I don't understand why your Qt designer acts different. When I add a vertical spacer, the height of tghe form is not changed at all - which is correct since it is a rubber length:
![designer_mSmuNADaET](https://user-images.githubusercontent.com/1828501/172501933-1ad9aa67-0ad5-4f90-849c-1f513cdfc4d0.gif)

- What is the problem with TaskBalloon.ui? Do you have a screenshot for me please?